src/patternfly/components/SimpleList/simple-list-item-link.hbs

```diff
@@ -1,4 +1,4 @@
-<{{#if simple-list-item-link--IsLink}}a{{else}}button{{/if}} class="{{pfv}}simple-list__item-link{{#if simple-list-item-link--modifier}} {{simple-list-item-link--modifier}}{{/if}}"
+<{{#if simple-list-item-link--IsLink}}a{{else}}button{{/if}} class="{{pfv}}simple-list__item-link{{#if simple-list-item-link--IsLink}} pf-m-link{{/if}}{{#if simple-list-item-link--modifier}} {{simple-list-item-link--modifier}}{{/if}}"
```

Comment:
🔥

FYI sometimes we'll write it this way just 'cause it's more readable. Note the `~`'s, which removes any whitespace (new line chars in this case) before it. Up to you if you want to update it.

```suggestion
<{{#if simple-list-item-link--IsLink}}a{{else}}button{{/if}} class="{{pfv}}simple-list__item-link
  {{~#if simple-list-item-link--IsLink}} pf-m-link{{/if}}
  {{~#if simple-list-item-link--modifier}} {{simple-list-item-link--modifier}}{{/if}}"
```